graphics.c

```diff
@@ -0,0 +1,158 @@
+#include "graphics.h"
+#include "display.h"
+
+static inline void vGraphicsPixelWrite(color pixelColor)
+{
+    vDisplayPixelWrite(pixelColor.colorSeperate.redValue, pixelColor.colorSeperate.greenValue, pixelColor.colorSeperate.blueValue);
+}
+
+
+curserStruct curserA,curserB;
+
+void graphicsInit()
+{
+
+vDisplayClear();    //blackout the display
+
+curserA.curserColor=ACURSERCOLOR;
+curserB.curserColor=BCURSERCOLOR;
+
+printGrid();    //print the grid
+
+}
+
+//printRectangle prints a Rectangle
+
+void printRectangle(uint16_t xStart,uint16_t xEnd,uint16_t yStart,uint16_t yEnd,color lineColor)
+{
+    uint16_t row,column;
+
+    vDisplayWindowSet(xStart,xEnd,yStart,yEnd);
+    vDisplayStartPixelWrite();
+
+    for(row=yStart;row<=yEnd;row++)
+    {
+        for(column=xStart;column<=xEnd;column++)
+        {
+            vGraphicsPixelWrite(lineColor);
+        }
+    }
+}
+
+
+//printLineBresenham prints a line between two points, including the second point, excluding the first point
+void printLineBresenham(uint16_t xA,uint16_t yA,uint16_t xB,uint16_t yB,color lineColor)
+{
+uint16_t x=xA,y=yA,dX=xB-xA,dY;
+int16_t delta,step=1;
+uint8_t risingLine=1;
```

Comment:
- Einrücken!
- Alle jeweils in eigene Zeile.
- `risingLine` umbenennen in `ucRisingLine` (uc = unsigned char = uint8_t)
- `delta` und `step` in `iDelta` und `iStep` (i = int16)
- `xA`, ... jeweils in `iX_A`, ... umbenennen